cmake/Findexpat.cmake

```diff
@@ -8,7 +8,12 @@ if(NOT TARGET expat::expat)
 
     if(EXPAT_FOUND)
         if(NOT TARGET expat::expat)
-            add_library(expat::expat ALIAS EXPAT::EXPAT)
+            if(${CMAKE_VERSION} VERSION_LESS 3.18)
```

Comment:
This might be a bit pedantic, but I think we should just drop the CMake 3.18+ behavior and only use the code that works on older CMakes.